gitlab/tests/test_integration.py

```diff
@@ -48,6 +47,26 @@ def test_connection_failure(aggregator, gitlab_check, bad_config):
         )
 
 
+@requires_py3
+def test_connection_failure_openmetrics(dd_run_check, aggregator, gitlab_check, bad_config):
```

Comment:
It looks like we're missing tests for the other service checks (to cover the `_check_health_endpoint` method).